### PR TITLE
feat(store): add several Australian stores, add some 3080 brands/models

### PR DESCRIPTION
**Pull request by kin3tik**
### Description

Added several Australian stores:
- BPCTech (AU)
- Centrecom (AU)
- CPL (AU)
- Mwave (AU)
- PCCG (AU)
- Scorptec (AU)
- Umart (AU)

Added some missing RTX 3080 brands/models:
- colorful (igame advanced oc, igame vulcan oc)
- galax (sg, sg oc)
- gigabyte (aorus xtreme waterforce wb)
- leadtek (hurricane)

### CODE REVIEW

**Review by jef — COMMENTED** [review]
Oh wow... Quite a lot of stores added! That's awesome. Thank you so much for your time spent on these!

It looks like there is just a small linting issue. Easiest way to fix that is to run `npm run lint:fix`.

Once this passes linting, we'll get this merged in.

Thank you very much!

**Comment by kin3tik** [comment]
Looks like a recent commit broke the stores. I'll add the currency prop.

File: src/store/model/scorptec.ts [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+	backoffStatusCodes: [403, 429],
+	currency: '$',
+	labels: {
+		inStock: {
+			container: '#delivery-wrapper > div.product-stock-text',
+			text: [' IN STOCK']
+		},
+		outOfStock: {
+			container: '#delivery-wrapper > div.product-stock-text',
+			text: [' SOLD OUT ']
```
Comment (jef) [inline src/store/model/scorptec.ts:13]:
```suggestion
			text: ['in stock']
		},
		outOfStock: {
			container: '#delivery-wrapper > div.product-stock-text',
			text: ['sold out']
```

File: src/store/model/umart.ts [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+			container: '#youhuo_delivery',
+			text: ['In Stock - for Delivery']
+		},
+		maxPrice: {
+			container: '.goods-price',
+			euroFormat: false
+		},
+		outOfStock: {
+			container: 'div.price-box > div.stock-label',
+			text: ['Out Of Stock - for Delivery']
```
Comment (jef) [inline src/store/model/umart.ts:17]:
```suggestion
			text: ['in stock']
		},
		maxPrice: {
			container: '.goods-price',
			euroFormat: false
		},
		outOfStock: {
			container: 'div.price-box > div.stock-label',
			text: ['out of stock']
```

File: src/store/model/bpctech.ts [hunk 3]
```diff
@@ -0,0 +1,171 @@
+import {Store} from './store';
+
+export const Bpctech: Store = {
+	backoffStatusCodes: [403, 429],
+	currency: '$',
+	labels: {
+		inStock: {
+			container: '.productStockStatus',
+			text: ['In Stock at BPC Tech']
```
Comment (jef) [inline src/store/model/bpctech.ts:9]:
```suggestion
			text: ['in stock']
```

**Review by jef — COMMENTED** [review]
OK! We're just about here. I think there could be some potential issues with the text in the labels, only because we don't do any trimming on the text.

We should, but not sure of the side effects.

Let me know what you think. I appreciate all the support!

**Comment by kin3tik** [comment]
Sounds fair. I wasn't sure how the container text was treated (is it just a contains?) so I was just using the result of innerText on the query selector verbatim.

**Comment by jef** [comment]
> Sounds fair. I wasn't sure how the container text was treated (is it just a contains?) so I was just using the result of innerText on the query selector verbatim.

Yup! It should _mostly_ work. I've sure it would've worked for the most part too. I have updated mostly just for consistency purpose now.

Thanks for working on this, it's a great addition! I'm sure there are some Aussie folks out there that can appreciate this!

**Review by jef — APPROVED** [review]

**Pull request merged**